net/connstats/stats.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package connstats maintains statistics about connections
+// flowing through a TUN device (which operate at the IP layer).
+package connstats
+
+import (
+	"net/netip"
+	"sync"
+
+	"tailscale.com/net/packet"
+	"tailscale.com/types/netlogtype"
+)
+
+// Statistics maintains counters for every connection.
+// All methods are safe for concurrent use.
+// The zero value is ready for use.
+type Statistics struct {
```

Comment:
GitHub is being silly, this is mostly identical to the removed `tunstats.Statististics` type.